services/build.gradle

```diff
@@ -12,7 +12,8 @@ sourceSets {
 dependencies {
     api project(":services-core")
     api project(":services-geojson")
-    api "com.mapbox.mapboxsdk:mapbox-sdk-directions-models:5.1.0-SNAPSHOT"
+    api project(":services-directions-models")
```

Comment:
I believe we should keep adding the explicit dependency until it's released. Same for `services-directions-refresh-models` cc @lukaspakzos

Reply:
yea, I think that's correct

Reply:
Do we still want to keep it as explicit dependency? 
@LukasPaczos  @Guardiola31337